core/storage/collection/gae_models_test.py

```diff
@@ -82,6 +85,36 @@ def test_get_all_non_private_commits(self):
         self.assertEqual(False, commits[2])
         self.assertEqual('collection--b--0', commits[0][0].id)
 
+    def test_get_all_non_private_commits_with_invalid_max_age(self):
+        with self.assertRaisesRegexp(
+            Exception,
+            'max_age must be a datetime.timedelta instance or None.'):
+            (
+                collection_models.CollectionCommitLogEntryModel
+                .get_all_non_private_commits(2, None, 'invalid_max_age'))
+
+    def test_get_all_non_private_commits_with_max_age(self):
+        private_commit = collection_models.CollectionCommitLogEntryModel.create(
+            '-a-', 0, 'commiter_id', 'username', 'msg',
```

Comment:
Nit: committer not commiter. Ditto below.

Reply:
Done